src/EditorFeatures/CSharpTest/Completion/CompletionProviders/CompletionProviderOrderTests.cs

```diff
@@ -70,6 +70,7 @@ public void TestCompletionProviderOrder()
             // Built-in interactive providers
             typeof(LoadDirectiveCompletionProvider),
             typeof(ReferenceDirectiveCompletionProvider),
+            typeof(ProjectDirectiveCompletionProvider),
```

Comment:
As per the prior meeting, i don't think this should be built into roslyn.  It should just use hte public API surface.

Reply:
It looks like just shipping an analyzer dll which has the completion provider types in it should cause it to get picked up, e.g. the following completion provider in AspNetCoreAnalyzers https://github.com/dotnet/aspnetcore/blob/8883b98afedee35e8360db3f24ce8cd2e17217ad/src/Framework/AspNetCoreAnalyzers/src/Analyzers/RouteEmbeddedLanguage/RoutePatternCompletionProvider.cs#L28